examples/Basic usage - Functional.ipynb

```diff
@@ -142,29 +144,40 @@
     "\n",
     "trial = 1\n",
     "\n",
-    "plt.figure(figsize=(12, 8))\n",
+    "plt.figure(figsize=(12, 4))\n",
     "\n",
     "for i_roi in range(n_roi):\n",
     "    plt.plot(result[i_roi, trial][0, :], label=\"ROI {}\".format(i_roi))\n",
     "\n",
     "plt.title(\"Trial {}\".format(trial), fontsize=15)\n",
-    "plt.xlabel(\"Frame number\", fontsize=15)\n",
-    "plt.ylabel(\"Signal\", fontsize=15)\n",
+    "plt.xlabel(\"Time (frame number)\", fontsize=15)\n",
+    "plt.ylabel(\"Signal intensity (candela per unit area)\", fontsize=15)\n",
     "plt.grid()\n",
     "plt.legend()\n",
     "plt.show()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We next plot the traces across all ROIs and trials:"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Fetch the colormap object for Cynthia Brewer's Paired color scheme\n",
+    "cmap = plt.get_cmap(\"Paired\")\n",
```

Comment:
I was intentionally not using this colormap for this notebook because there is only the result line and no raw line to show. I did not want to needlessly add to the complexity of what is going on when it doesn't make the plots any clearer.

This change would make sense in #237, but not here.

Reply:
I think it looks much better this way, especially since we used the color code in for different ROIs before. Also, if you want this to be all blue than they should also be all blue in the other note-books, just have different shades for raw or separated. The colors don't really help with that. For me the colors are nice as it's immediately clear that different traces come from different cells.

Reply:
But I'm happy with both. I just figured you had not gone back to the functional notebook to update it with the nice plotting code from the other notebooks =).

Reply:
In any case with #237 this will all be moot.